README.md

```diff
@@ -726,11 +722,11 @@ Assuming we called this `dashboards.py` we could run it in one of two ways:
   - Give the script execution rights and run it directly
   (typically `chmod +x dashboards.py`)
       - `./dashboards.py --api-key mykey update`
-  - Pass the script in to the Python executor
+  - Pass the script into the Python executor
       - `python dashboards.py --api-key mykey update`
 
-If you want to know about the available actions you can take with your new
-CLI you can always the `--help` command.
+If you want to know about the available actions, you can take with your new
```

Comment:
I think this comma should not be added.

Reply:
I'd consider it an introductory clause warranting a comma.

https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_after_introductions.html#:~:text=Introductory%20clauses%20are%20dependent%20clauses,athletes%20must%20exercise%20every%20day.

Reply:
The problem is that  "you can take with your new CLI" is not an independent clause.  The "with" is meaningless without the first part, because the phrase is actually "the available actions you can take with your new CLI" and is the object of the if clause "If you want to know about...".

It's really the whole thing up to "CLI" that is the introductory clause; the independent clause is "you can always use the  --help command".

Reply:
@jrduncans Ah okay, that makes sense. I appreciate the thorough explanation. I am reverting this change.